tools/dotnet-deb-tool/project.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0-rc3-004308"
+      "version": "1.0.0"
     }
   },
```

Comment:
No need to run against latest here. Again this needs to run against the shared fx in the sdk.